annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java

```diff
@@ -396,6 +397,14 @@ public void setGNOMAD_SAS_AF(String gnomadAlleleFrequencySAS) {
         this.gnomadAlleleFrequencySAS = gnomadAlleleFrequencySAS;
     }
 
+    public String getANNOTATION_STATUS() {
+        return this.annotationStatus != null ? this.annotationStatus : "FAILED";
```

Comment:
wouldn't it be cleaner to always set the annotationStatus when we attempt annotation? (see my comment above for the HTTP GET) ... if this.annotationStatus is null, that could be interpreted as meaning "we did not attempt to annotated this variant" rather than assuming it means the annotation failed. By interpreting null as meaning FAILED, we lose that distinction.